gate-core/src/main/java/com/netflix/spinnaker/gate/config/TaskServiceProperties.java

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright 2022 Netflix, Inc.
```

Comment:
```suggestion
 * Copyright 2022 Armory, Inc.
```
?

Reply:
This comes from... spotless I believe.  Can change it, but not that worried about this one...